### PR TITLE
Sixfab shield

### PR DESCRIPTION
**Pull request by Worthless-Inventions**
It seems that I missed this board out before so I have done it now. 

Hope this one and the adafruit ones from earlier are alright

### CODE REVIEW

File: draft/overlay/sixfab_GSM:GPRS.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+---
+name: GSM/GPRS Shield
+class: board
+type: com
+formfactor: pHAT
+manufacturer: Sixfab
+description: Connect from anywhere to the internet on a Raspberry Pi
+url: http://sixfab.com/product/gsmgprs-shield/
+buy: http://sixfab.com/product/gsmgprs-shield/
+image: 'sixfab-gsmgprs-shield.png'
```
Comment (Gadgetoid) [inline draft/overlay/sixfab_GSM:GPRS.md]: Should match the filename of the supplied image.

**Comment by Gadgetoid** [comment]
Could you rename from `sixfab_GSM:GPRS.md` to `sixfab-GSM-GPRS.md` or similar. The `:` character and Git Bash on Windows aren't the best of friends- resulting in empty files.

**Comment by Worthless-Inventions** [comment]
That should do it. Sorry for the error! Do you know when the adafruit boards I did that are in the draft folder will be pushed onto the website or is there an error I need to fix?

**Comment by Gadgetoid** [comment]
I'm not sure about the adafruit boards, I'll look into them.

**Comment by Gadgetoid** [comment]
Ooh another issue:

```text
Traceback (most recent call last):
  File "./generate-html.py", line 554, in <module>
    overlays = map(load_overlay, overlays)
  File "./generate-html.py", line 190, in load_overlay
    "{} includes a reference to a {} pin ({}), which isn't allowed".format(overlay, actual_pin['type'], pin))
Exception: sixfab-gsmgprs-shield includes a reference to a +5v pin (4), which isn't allowed
```

I think this might be because you're using BCM pin numbering in your overlay file- and physical pin 4 is 5v, you'll need to switch all of the pins over to the corresponding *physical* pin numbers. (It's possible to prefix them with "bcm" iirc, but physical is preferred for consistency)

Thank you!

**Comment by Worthless-Inventions** [comment]
Hopefully that is right.

**Comment by Gadgetoid** [comment]
That seems to have done the trick! I can't verify the accuracy of the pins selected, but it built without error.

**Pull request merged**